assets/app/views/_pod-template.html

```diff
@@ -11,10 +11,10 @@
       <div class="pod-template-image">
         <span class="fa fa-cube" aria-hidden="true"></span>
         <span>Image: {{container.image | imageStreamName}}
```

Comment:
what does this look like if `container.image` is missing? should the `pod-template-image` block be conditional on that?